src/components/FilterHeader.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { documentTitle } from '../common/variables';
+import { Grid, Button } from '@material-ui/core';
+import IconFilter from '@material-ui/icons/FilterList';
+import { CapturesContext } from '../context/CapturesContext';
+import Navbar from '../components/Navbar';
+import FilterTop from '../components/FilterTop';
+
+function FilterHeader() {
+  const capturesContext = useContext(CapturesContext);
+  const [isFilterShown, setFilterShown] = useState(true);
+
+  useEffect(() => {
+    document.title = `Capture Data - ${documentTitle}`;
```

Comment:
I think this should be in `CapturesView`

Reply:
The useEffect for the documentTitle and the filter state? 
Or should I spread out the code for the Nav and FilterTop in the FilterHeader in the CaptureView instead of creating a new component?  Then I could rename FilterTop to CaptureFilter and we could use the same nesting for the Growers with a GrowersView and GrowersFilter?

Reply:
Just the doc title. We should be setting it in the view components, as they're the top level at which the title applies.
The additional refactoring you suggest does make sense, but could probably wait for now.